lib/shelley/test/unit/Cardano/Wallet/Shelley/TransactionSpec.hs

```diff
@@ -3842,7 +3846,10 @@ mockProtocolParametersForBalancing = (mockProtocolParameters, nodePParams)
         , Cardano.protocolParamPoolPledgeInfluence = 0
         , Cardano.protocolParamMonetaryExpansion = 0
         , Cardano.protocolParamTreasuryCut  = 0
-        , Cardano.protocolParamUTxOCostPerWord = Just 34482
+        , Cardano.protocolParamUTxOCostPerWord =
+            Just $ fromIntegral $ SL.unCoin testParameter_coinsPerUTxOWord_Alonzo
```

Comment:
It might be possible to use `intCast` here, which checks statically that the conversion is lossless.